.github/workflows/test.yml

```
@@ -0,0 +1,31 @@
+name: Run Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - develop
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

May as well use the latest version.
```suggestion
      - uses: actions/checkout@v5
```